src/main/java/org/broadinstitute/consent/http/mail/MailService.java

```diff
@@ -136,4 +137,10 @@ public void sendNewHelpReportMessage(Set<String> usersAddress,  Writer template,
         sendMessages(messages);
     }
 
-}
\ No newline at end of file
+    @Override
+    public void sendNewResearcherApprovedMessage(Set<String> researcherEmails, Writer template, String darCode) throws MessagingException {
```

Comment:
Could use some unit testing to ensure that it runs successfully and one for failing with a `MessagingException`